funcake_dags/scripts/csv_to_xml.py

```diff
@@ -10,20 +10,31 @@
 TIMESTAMP = os.environ.get("TIMESTAMP")
```

Comment:
It doesn't appear that you are using this anymore.  Should it be removed?

Reply:
@sensei100 I'm passing in the timestamp when I called the method
https://github.com/tulibraries/funcake_dags/blob/538007f21db22d488caca886c8f7c87e437ceb09/funcake_dags/scripts/csv_to_xml.py#L35